buildpack.toml

```diff
@@ -16,19 +16,19 @@ api = "0.7"
   [[metadata.dependencies]]
     id = "miniconda3"
     name = "Miniconda"
-    sha256 = "536817d1b14cb1ada88900f5be51ce0a5e042bae178b5550e62f61e223deae7c"
-    source = "https://github.com/conda/conda/archive/4.9.2.tar.gz"
-    source_sha256 = "d7f946e5c770e45da8961323ca96399bf1a881eb68bbaecc7cb1e249f5c86d54"
+    sha256 = "4ee9c3aa53329cd7a63b49877c0babb49b19b7e5af29807b793a76bdb1d362b4"
+    source = "https://github.com/conda/conda/archive/4.11.0.tar.gz"
+    source_sha256 = "1843355ccb93afb05f2a307e1fc37403fb9976da799236eebc3adee1c716c5fc"
     stacks = ["io.buildpacks.stacks.bionic", "org.cloudfoundry.stacks.cflinuxfs3"]
-    uri = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
-    version = "4.9.2"
-    cpe = "cpe:2.3:a:conda:miniconda3:4.9.2:*:*:*:*:python:*:*"
+    uri = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.11.0-Linux-x86_64.sh"
+    version = "4.11.0"
+    cpe = "cpe:2.3:a:conda:miniconda3:4.11.0:*:*:*:*:python:*:*"
 
   [[metadata.dependency_deprecation_dates]]
```

Comment:
I'd suggest we remove the `dependency_deprecation_dates` section entirely, as I don't think the provided URI actually contains deprecation dates for `miniconda` - it refers to the deprecation policy for `python 3.9` and I cannot find any evidence that this policy applies to `miniconda` too.